eng/pipelines/pull-request-common.yml

```diff
@@ -55,16 +55,15 @@ steps:
 
   - script: node common/scripts/install-run-rush.js check-format
     displayName: Check Formatting
+    condition: ne(variables['Agent.OS'], 'Windows_NT')
 
   - script: node common/scripts/install-run-rush.js lint --verbose
     displayName: Lint
+    condition: ne(variables['Agent.OS'], 'Windows_NT')
 
   - script: cd packages/samples && npm run regen-samples
     displayName: Regenerate Samples
 
-  - script: node common/scripts/install-run-rush.js regen-docs
-    displayName: Regenerate Reference Docs
```

Comment:
Don't we want to still do this on linux?

Reply:
I think this was purely redundant. As far as I can tell this reruns the regen-ref-docs that the website build also does. @allenjzhang Am I missing something?